app/src/main/java/com/devshish/internship/presentation/ui/service/client/MediaBrowserClient.kt

```diff
@@ -39,6 +43,15 @@ class MediaBrowserClient(context: Context) {
         )
     }
 
+    fun setSong(song: Song) {
+        songToPlay = song
+    }
+
+    fun getSong(): Song? = songToPlay
+
+    fun playFromUri(): Unit =
+        mediaController.transportControls.playFromUri(songToPlay?.uri, null)
```

Comment:
same here - make method like playSong(song) instead of 2 methods setSong+playFromUri